{
  "totalNumberSkippingTests": 0,
  "bugType": "failing_passing",
  "totalNumberErroringTests": 1,
  "repo": "wix/wix-embedded-mysql",
  "failingModule": "/root/workspace/wix/wix-embedded-mysql/239549452",
  "hostname": "repairnator",
  "totalNumberFailingTests": 0,
  "error-types": [
    "java.lang.RuntimeException"
  ],
  "totalNumberRunningTests": 1,
  "failing-test-cases": [
    {
      "className": "com.wix.mysql.EmbeddedMysqlTest",
      "failingMethods": [],
      "erroringMethods": [
        "com.wix.mysql.EmbeddedMysqlTest#EmbeddedMysql instance should::start with default values"
      ],
      "failures": [
        {
          "failureName": "java.lang.RuntimeException",
          "failureDetail": "Command exited with error code: \u0027127\u0027 and output: \u0027\u0027",
          "isError": true
        }
      ],
      "nbFailures": 0,
      "nbErrors": 1
    }
  ]
}